test/user/user_db_acl_test.go

```diff
@@ -0,0 +1,192 @@
+// Copyright IBM Corp. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package user
+
+import (
+	"io/ioutil"
+	"net/http"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/hyperledger-labs/orion-server/pkg/constants"
+	"github.com/hyperledger-labs/orion-server/pkg/server/testutils"
+	"github.com/hyperledger-labs/orion-server/pkg/types"
+	"github.com/hyperledger-labs/orion-server/test/setup"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+)
+
+var baseNodePort uint32
+var basePeerPort uint32
+var portMutex sync.Mutex
+
+func init() {
+	baseNodePort = 6100
+	basePeerPort = 7100
+}
+
+func getPorts(num uint32) (node uint32, peer uint32) {
+	portMutex.Lock()
+	defer portMutex.Unlock()
+
+	node = baseNodePort
+	peer = basePeerPort
+	baseNodePort += num
+	basePeerPort += num
+
+	return
+}
+
+// Scenario:
+func TestUserACLOnDatabase(t *testing.T) {
+	dir, err := ioutil.TempDir("", "int-test")
+	require.NoError(t, err)
+
+	nPort, pPort := getPorts(1)
+	setupConfig := &setup.Config{
+		NumberOfServers:     1,
+		TestDirAbsolutePath: dir,
+		BDBBinaryPath:       "../../bin/bdb",
+		CmdTimeout:          10 * time.Second,
+		BaseNodePort:        nPort,
+		BasePeerPort:        pPort,
+		CheckRedirectFunc: func(req *http.Request, via []*http.Request) error {
+			return errors.Errorf("Redirect blocked in test client: url: '%s', referrer: '%s', #via: %d", req.URL, req.Referer(), len(via))
+		},
```

Comment:
when using one server you can leave this as nil `CheckRedirectFunc: nil`

Reply:
Done